src/viam/services/vision/client.py

```diff
@@ -253,7 +125,6 @@ async def get_classifications(
 
         Args:
             image (Image): The image to get detections from
-            classifier_name (str): The name of the detector to use for detection
 
```

Comment:
It looks like get_classifications is missing the "count" (also known as "n") argument from the api:
https://github.com/viamrobotics/api/blob/RSDK-2578-new-vision-service/proto/viam/service/vision/v1/vision.proto#L115

get_classifications_from_camera has it, though

Reply:
fixed!